I'm getting a massive list of errors from cmake when trying to build the LLVM part of this.  Did something major change in the 5 => 6 build setup/requirements?  (AFAICS it is strictly related to the LLVM 6 upgrade.)

```
CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-all" does not exist.
Call Stack (most recent call first):
  CMakeLists.txt:919 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-all" does not exist.
Call Stack (most recent call first):
  CMakeLists.txt:919 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-builtins" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/builtins/CMakeLists.txt:39 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-builtins" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/builtins/CMakeLists.txt:39 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-interception" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/interception/CMakeLists.txt:14 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-interception" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/interception/CMakeLists.txt:14 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-lsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/lsan/CMakeLists.txt:36 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-lsan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/lsan/CMakeLists.txt:36 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-ubsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/ubsan/CMakeLists.txt:71 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-ubsan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/ubsan/CMakeLists.txt:71 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-cfi" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/cfi/CMakeLists.txt:88 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-cfi" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/cfi/CMakeLists.txt:88 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-cfi-and-supported" does not
  exist.
Call Stack (most recent call first):
  projects/compiler-rt/test/cfi/CMakeLists.txt:92 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-cfi-and-supported"
  does not exist.
Call Stack (most recent call first):
  projects/compiler-rt/test/cfi/CMakeLists.txt:92 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-sanitizer" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/sanitizer_common/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-sanitizer" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/sanitizer_common/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/fuzzer/CMakeLists.txt:40 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-fuzzer" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/fuzzer/CMakeLists.txt:40 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-asan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/asan/CMakeLists.txt:159 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-asan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/asan/CMakeLists.txt:159 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-asan-dynamic" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/asan/CMakeLists.txt:165 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-asan-dynamic" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/asan/CMakeLists.txt:165 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-dfsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/dfsan/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-dfsan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/dfsan/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-msan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/msan/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-msan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/msan/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-tsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/tsan/CMakeLists.txt:100 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-tsan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/tsan/CMakeLists.txt:100 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-safestack" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/safestack/CMakeLists.txt:26 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-safestack" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/safestack/CMakeLists.txt:26 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-esan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/esan/CMakeLists.txt:29 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-esan" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/esan/CMakeLists.txt:29 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-scudo" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/scudo/CMakeLists.txt:38 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-scudo" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/scudo/CMakeLists.txt:38 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-ubsan-minimal" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/ubsan_minimal/CMakeLists.txt:23 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-ubsan-minimal" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/ubsan_minimal/CMakeLists.txt:23 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-profile" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/profile/CMakeLists.txt:26 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-profile" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/profile/CMakeLists.txt:26 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang" of target "check-xray" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/xray/CMakeLists.txt:39 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1333 (add_dependencies):
  The dependency target "clang-headers" of target "check-xray" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1354 (add_lit_target)
  projects/compiler-rt/test/xray/CMakeLists.txt:39 (add_lit_testsuite)
```

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -65,7 +65,7 @@ parts:
 
   ldc-bootstrap:
     source: https://github.com/ldc-developers/ldc.git
-    source-tag: v0.17.5
+    source-tag: v0.17.6
     source-type: git
     plugin: cmake
     configflags:
@@ -87,11 +87,12 @@ parts:
 
   llvm:
     source: https://github.com/ldc-developers/llvm.git
-    source-tag: ldc-v5.0.1
+    source-tag: ldc-v6.0.1-3
     source-type: git
     plugin: cmake
     configflags:
     - -DCMAKE_BUILD_TYPE=Release
+    - -DCOMPILER_RT_INCLUDE_TESTS=OFF
     - -DLLVM_BINUTILS_INCDIR=/usr/include
     - -DLLVM_TARGETS_TO_BUILD=X86\;AArch64\;ARM\;PowerPC\;NVPTX
     stage:

